### ISSUE TITLE
Declining sample size in weekly contingency tables

### ISSUE THREAD

**Issue by nmdefries** [cmu-delphi/covidcast-indicators#874]
**Actual Behavior:**

[Weekly contingency tables](https://cmu.app.box.com/s/xwjulq0pteen52d4upni9ikagu7d8bl2) generated from Symptom Survey data show declining sample size in pct_vaccinated indicator over the period Jan 10-Feb 21 (6 weeks). Looking at `<date>_nation_65orolder_hispanic_gender_race.csv` files for a range of dates, we see

```
sum(Jan10$sample_size_pct_vaccinated)
sum(Jan17$sample_size_pct_vaccinated)
sum(Jan31$sample_size_pct_vaccinated)
sum(Feb14$sample_size_pct_vaccinated)
```

![Screen Shot 2021-02-26 at 18 44 00 ](https://user-images.githubusercontent.com/42820733/109366903-9d880f80-7862-11eb-8b7b-580203aa1cfb.png)

**Expected behavior**

We expect a relatively stable ~350k survey responses per week (~50k per day), with something like 20% removed for not answering the particular question at hand, resulting in a sample size of 280k for pct_vaccinated calculations each week.

**Context**

May be related to issue #873 , although the drop in sample size should only impact tables for the week of Jan 31.

In the sample size plot in issue #873 , we can see that there has been a steady decline in daily sample size, from 50k per day to 40k (-20%) over the course of a month. While this may be contributing to the contingency table issue, from the week of Jan 10 to the week of Feb 14 we actually see a drop in sample size of nearly 40%.

**Comment by nmdefries** [comment]
tl;dr: Week of Feb14 was erroneously using a partial week of data. Normally except to see a sample size of 230k used for calculating vaccination rate.

Looking at raw data, the week of Jan 31 has 364k responses (362k consenting, 361k consenting and answered item V1). The week of Feb 14 has similar numbers of responses.

Standard filtering (known zip code, reasonable number of household members, # sick household members < total number, etc) removes about 25% of responses -> 278k. Vaccine question (V1) is missing with 5% rate; gender is missing at a higher-than-usual rate -- 16% -> 231k responses.

This seems pretty consistent over the last several weeks. The beginning of January stands out as having more responses (around 260k available for use in calculating vaccination rate), maybe a holiday effect (more time to do surveys)? This is also apparent in sample sizes used for raw_cli in #873 .
